src/bearhug.js

```diff
@@ -117,7 +117,6 @@ angular.module('talis.bearhug', [])
                     if (!oauthTokenRetryFail && response && response.status === 401 && response.data &&
```

Comment:
This ''invalid_token' and 'expired_token' stuff is also Persona specific.  For Talis use that's fine, for external use not so.